.github/workflows/ci_pipeline.yml

```diff
@@ -82,6 +87,73 @@ jobs:
             data_3.10
             coverage_${{ matrix.env }}_3.10.dat
 
+  project_cpu_tests_linux:
+    runs-on: ubuntu-latest
+
+    strategy:
+      fail-fast: false
+      matrix:
+        env: 
+          - AllenCahn_Bayreuth
+          - AsympConv
+          - DAE
+          - FastWaveSlowWave
+          # - GPU
+          - Hamiltonian
+          - matrixPFASST
+          - parallelSDC
+          - parallelSDC_reloaded
+          - PinTSimE
+          - RDC
+          - Resilience
+          - SDC_showdown
+          - Second_orderSDC
+          - soft_failure
+          - TOMS
+        # python: ['3.8', '3.9', '3.10']
+        python: ['3.10']
```

Comment:
@pancetta also wants all python versions tested here. Btw: Did you test 3.11 and 3.12? Especially with the non-project tests?

Reply:
I forgot what the issue was. That's why I wanted to test everything with everything and see if the base stuff at least works with the latest python. That would be nice because I would like to transition to a newer version on my laptop. On the cluster I have been using 3.11 with no issues. Actually, the HPC tests are probably all run with 3.11, maybe..?

Reply:
No, I did not test more python versions. This reduction on python-versions is also only for current development, as the CI is faster with fewer jobs and finds most of the errors also with only a single python-version.

It is not meant to stay like that for merging.

Reply:
Reverted in eebbdd3b89a71f3c3063efccfab926abde748fea and also added more python-versions for testing (3.11 & 3.12)